Overview.bs

```diff
@@ -501,9 +504,10 @@ The algorithm:
 11. Go to step 2.
 
 
-Note: in step 9 the client may optionally start loads for patches which are not invalidated by the currently selected
-entry. This is an important optimization which significantly improves performance by eliminating network round trips
-where possible by initiating loads for patches that will be needed in later iterations.
+Note: in step 9 the client may optionally start loads in parallel for patches which are not invalidated by the currently
+selected entry. This is an important optimization which significantly improves performance by eliminating network round
```

Comment:
This sentence might be better as (with the parenthetical clarified by you):

> Such patches will (almost always?) be needed in later iterations, and loading them in parallel significantly improves performance by eliminating network round-trips.

Reply:
Sounds good, updated. Almost always is pretty accurate assessment, it's technically possible for a partial invalidating patch to change the non-invalidating mappings without invalidating the currently listed patches, but I wouldn't expect any reasonable IFT encoding to do this.